config/components/crd/bases/kueue.x-k8s.io_clusterqueues.yaml

```diff
@@ -12,8 +12,6 @@ spec:
     kind: ClusterQueue
     listKind: ClusterQueueList
     plural: clusterqueues
-    shortNames:
-    - cq
```

Comment:
Is the removal of `shortName` intended?

Reply:
yeah.... too cryptic

Reply:
It sounds good to me.
In this case, we may need to remove kubebuiler markers from v1alpha1 API.